### ISSUE TITLE
Incorrect suggested level

### ISSUE THREAD

**Issue by langerest** [jason-whitted/gloomy#29]
The suggested level should be average character level/2 ROUND UP.

**Comment by jason-whitted** [comment]
Precisely!

```js
const avgLevel = attendees.reduce((t, c) => t + c.level, 0) / attendees.length;
const suggestedLevel = Math.round(avgLevel / 2);
```

`avgLevel` is the sum of the attendees levels divided by the number of attendees.
Then `suggestedLevel` is the `avgLevel` divided by 2 and rounded up.

Are you seeing something different?

**Comment by langerest** [comment]
Today we have a group of lv 7, 7, 7, 6, 6. The suggested level should be 4 but gloomy gave a suggested level 3. Are you sure Math.round() does round up?

**Comment by langerest** [comment]
Checked some javascript manual. You should use Math.ceil() here.

**Comment by jason-whitted** [comment]
You are absolutely right!  I've never even read the footnote on that page in the rule book.

>  4.25; divided by 2 and rounded up is 3

Every programmer on earth is rolling their eyes though.  That is **not** rounding up!  That's a ceil operation.

Although I admit that would confuse all of the non-Computer Science people.  :smile:

I'll update this logic and get it pushed with my evening update here in a few minutes.

**Comment by jason-whitted** [comment]
This has been fixed in 0.1.10
